src/sentry/api/endpoints/project_rules.py

```diff
@@ -291,20 +291,6 @@ class ProjectRulesPostSerializer(serializers.Serializer):
 }
 ```
 
-**Create an Azure DevOps work item**
```

Comment:
Moving this to the bottom b/c I assume Github is much more popular than azure devops